api/postman/bookings-env-postman.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I'm also guessing that this file shouldn't be part of this commit / PR?

Reply:
No I updated the file to include the new field I added.